cli/derive/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+//! Crate with a proc macro for torii endpoint generation
+
+use proc_macro::TokenStream;
+use proc_macro2::Span;
+use quote::quote;
+use syn::{
+    parse::{Parse, ParseStream},
+    parse_macro_input, Ident, LitInt, Result as SynResult,
+};
+
+/// Generates warp filters for endpoints accepting functions
+/// from 2 up to `n` arguments inclusively. Currently accepted
+/// maximum is 8. Also relies on `WarpResult` custom wrapper,
```

Comment:
I'd suggest rewriting it like this:
```suggestion
/// Generates warp filters for endpoints, accepting functions
/// with the number of arguments from 2 to 8 (inclusive, current maximum).
/// Also relies on `WarpResult` custom wrapper,
```